### ISSUE TITLE
NLUG is migrating its IRC presence to Libera Chat

### ISSUE THREAD

**Issue by rwanyoike** [nairobilug/nairobilug.or.ke#193]
Given the fiasco happening on Freenode ([1](https://fuchsnet.ch/freenode-resign-letter.txt)), with the acquisition and transfer of control, it would be best if our `#nairobilug` IRC channel were moved to `irc.libera.chat` along with bots.

**Comment by rwanyoike** [comment]
Updated the `Group description` and `Welcome message` on our mailing list https://groups.google.com/g/nairobi-gnu

> Nairobi GNU/Linux User Group | Kenya | Website: https://nairobilug.or.ke | Twitter: https://twitter.com/nairobilug | IRC: #nairobilug on Libera Chat | We traditionally meet on the first Saturday of every month | Karibu (Welcome)

**Comment by adrianteri** [comment]
Good afternoon @rwanyoike,  nlug_matterbot seems not to be configured yet (and possibly re-deployed?).

**Comment by rwanyoike** [comment]
Yes. I've redeployed `nlug_matterbot`. It should be relaying chats on `irc.libera.chat#nairobilug` now 👍🏽 .

**Comment by jnduli** [comment]
We'd also need to update the irc link in the website to point to liberachat too.

**Comment by rwanyoike** [comment]
Who owns the libera.chat #nairobilug channel now? We'll need `op` mode to setup `/topic`.

**Comment by fredmanglis** [comment]
What text would you like on the channel?

I have added the following in the meantime:

```
Welcome to #nairobilug. Check out our blog at https://nairobilug.or.ke/
```

**Comment by fredmanglis** [comment]
After looking around, I have copied over the topic set by @rwanyoike on Telegram and used it as the irc topic too.

For  convenience, the text is:

```
Nairobi GNU/Linux User Group | Kenya | Website: https://nairobilug.or.ke | Twitter: https://twitter.com/nairobilug | IRC: #nairobilug on Libera Chat | Discord: https://discord.com/invite/pxbFbdC | Telegram: https://t.me/nairobilug | We traditionally meet on the first Saturday of every month | Karibu (Welcome)
```
